apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -112,13 +113,14 @@ class LessonEditor extends Component {
     };
   }
 
-  handleSaveAndKeepEditing = e => {
+  handleSave = e => {
+    const isSaveAndClose = e.target.className.includes('btn-primary');
```

Comment:
sorry Dani, I am not sure if you are ready for comments yet, but this line makes me a little nervous. can you please pass in (maybe bind) a specific param indicating whether we plan to "save and close" rather than inferring it from the CSS class?

Reply:
I was just waiting to merge the previous PR before requesting feedback on this one. Its merged now.

I will update this.